R/rlistings.R

```diff
@@ -174,12 +177,21 @@ as_listing <- function(df,
 
   df <- as_tibble(df)
   varlabs <- var_labels(df, fill = TRUE)
-  o <- do.call(order, df[key_cols])
-  if (is.unsorted(o)) {
-    if (interactive()) {
-      message("sorting incoming data by key columns")
+  if (!is.null(sort_cols)) {
+    o <- do.call(order, df[sort_cols])
```

Comment:
I think we need to check if the sort columns are present

Reply:
Check added :)

Reply:
Thanks!!